vital/tests/test_image.cxx

```diff
@@ -297,6 +370,12 @@ TEST(image, equal_content)
     }
   }
   EXPECT_TRUE( equal_content( img1, img2 ) );
+
+  image img3( w, h, d, false, image_pixel_traits{ image_pixel_traits::BOOL } );
+  EXPECT_FALSE( equal_content( img1, img3 ) );
+
+  *img2.first_pixel()=1;
```

Comment:
better syntax would be `img2.at<bool>(0,0) = 1;`.  `first_pixel()` returns a `void*`.  I'm surprised this works without the compiler complaining.

Reply:
I believe this is actually calling the `image_of` class's `first_pixel()` function, which returns T* not void*, which is why the compiler is okay with it.